site/docs/spark.md

```diff
@@ -619,13 +619,30 @@ To show a table's data files and each file's metadata, run:
+| 0       | s3:/.../table/data/00000-3-8d6d60e8-d427-4809-bcf0-f5d45a4aad96.parquet | PARQUET     | 1            | 597                | [1 -> 90, 2 -> 62] | [1 -> 1, 2 -> 1] | [1 -> 0, 2 -> 0]  | [1 -> , 2 -> c] | [1 -> , 2 -> c] | null         | [4]           | null         |
+| 0       | s3:/.../table/data/00001-4-8d6d60e8-d427-4809-bcf0-f5d45a4aad96.parquet | PARQUET     | 1            | 597                | [1 -> 90, 2 -> 62] | [1 -> 1, 2 -> 1] | [1 -> 0, 2 -> 0]  | [1 -> , 2 -> b] | [1 -> , 2 -> b] | null         | [4]           | null         |
+| 0       | s3:/.../table/data/00002-5-8d6d60e8-d427-4809-bcf0-f5d45a4aad96.parquet | PARQUET     | 1            | 597                | [1 -> 90, 2 -> 62] | [1 -> 1, 2 -> 1] | [1 -> 0, 2 -> 0]  | [1 -> , 2 -> a] | [1 -> , 2 -> a] | null         | [4]           | null         |
++-----------------------------------------------------------------------------------+-------------+--------------+--------------------+--------------------+------------------+-------------------+-----------------+-----------------+--------------+---------------+--------------+
+```
+
+### Entries
+
+To show a table's manifest entries, run:
+
+```sql
+SELECT * FROM prod.db.table.entries
+```
+```text
++--------+----------------+-----------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| status | snapshot_id    | sequence_number | data_file                                                                                                                                                                                                |
++--------+----------------+-----------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 1      | 57897183625154 | 0               | [0, s3:/.../table/data/00000-3-8d6d60e8-d427-4809-bcf0-f5d45a4aad96.parquet, PARQUET, 1, 597, [1 -> 90, 2 -> 62], [1 -> 1, 2 -> 1], [1 -> 0, 2 -> 0], [1 -> , 2 -> c], [1 -> , 2 -> c], null, [4], null] |
+| 1      | 57897183625154 | 0               | [0, s3:/.../table/data/00001-4-8d6d60e8-d427-4809-bcf0-f5d45a4aad96.parquet, PARQUET, 1, 597, [1 -> 90, 2 -> 62], [1 -> 1, 2 -> 1], [1 -> 0, 2 -> 0], [1 -> , 2 -> b], [1 -> , 2 -> b], null, [4], null] |
+| 1      | 57897183625154 | 0               | [0, s3:/.../table/data/00002-5-8d6d60e8-d427-4809-bcf0-f5d45a4aad96.parquet, PARQUET, 1, 597, [1 -> 90, 2 -> 62], [1 -> 1, 2 -> 1], [1 -> 0, 2 -> 0], [1 -> , 2 -> a], [1 -> , 2 -> a], null, [4], null] |
++--------+----------------+-----------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
I had originally omitted this metadata table on purpose. We don't expose `ManifestEntry` in the public API because it is confusing for people to use and I would expect it to lead to bugs in other libraries. While it is nice to be able to inspect this, similar problems exist for the entries table. That's why I didn't call out support for it initially.

Reply:
I think that makes sense. As I mentioned in #1365, I wasn't sure whether this was intentional, so I just went ahead and added it. I can remove this. I have found the `entries` table to be really useful for looking at the set of files that have come between two snapshots (or in a set of snapshots).

Reply:
Yeah, let's either remove it or add just a note that says it exists, but is not recommended.